src/main/java/de/ait/secondlife/repositories/OfferRepository.java

```diff
@@ -14,4 +17,9 @@ public interface OfferRepository extends JpaRepository<Offer, Long> {
     Optional<Offer> findByIdAndIsActiveTrue(Long offerId);
 
     Page<Offer> findByUserIdAndIsActiveTrue(Long id, Pageable pageable);
+
+    List<Offer> findByAuctionFinishedAtLessThanEqualAndStatusNameAndIsActiveTrue(
```

Comment:
Very horrible method name. Need to think about rename, highly recommended

Reply:
in fact, this is not a name, but an instruction for spring data.
here we need to choose: either a stupid method name or write an additional sql-query with an annotation